docs/source/tutorials/06_using_your_own_models.rst

```diff
@@ -0,0 +1,578 @@
+*********************
+Using Your Own Models
+*********************
+
+.. include:: /include/substitution.rst
+
+PeekingDuck offers pre-trained :mod:`model` nodes that can be used to tackle a wide variety of
+problems, but you may need to train your own model on a custom dataset sometimes. This tutorial
+will show you how to package your model into a custom :mod:`model` node, and use it with
+PeekingDuck. We will be tackling a manufacturing use case here - classifying images of steel
+castings into "defective" or "normal" classes.
+
+Casting is a manufacturing process in which a material such as metal in liquid form is poured into
+a mold and allowed to solidify. The solidified result is also called a casting. Sometimes,
+defective castings are produced, and quality assurance departments are responsible for preventing
+defective pieces from being used downstream. As inspections are usually done manually, this adds
+a significant amount of time and cost, and thus there is an incentive to automate this process.
+
+The images of castings used in this tutorial are the front faces of steel `pump impellers 
+<https://en.wikipedia.org/wiki/Impeller>`_. From the comparison below, it can be seen that the
+defective casting has a rough, uneven edges while the normal casting has smooth edges.
+
+   .. figure:: /assets/tutorials/normal_vs_defective.png
+      :width: 416
+      :alt: Normal casting compared to defective casting
+
+      Normal Casting Compared to Defective Casting
+
+
+.. _tutorial_model_training:
+
+Model Training
+==============
+
+PeekingDuck is designed for model *inference* rather than model *training*. This optional section
+shows how a simple Convolutional Neural Network (CNN) model can be trained separately from the
+PeekingDuck framework. If you have already trained your own model, the
+:ref:`following section <tutorial_using_your_trained_model_with_peekingduck>` describes how you can
+convert it to a custom model node, and use PeekingDuck for inference.
```

Comment:
Suggest to phrase as "..., and use it within PeekingDuck for inference."

Reply:
Rephrased.